docs/how-to-guides/how-to-create-maps-for-autoware.md

```diff
@@ -0,0 +1,64 @@
+
+If you prefer proprietary software that is easy to use, you can try a fully automatic mapping tool from [MAP IV, Inc.](https://www.map4.jp/), [_MapIV Engine_](https://www.map4.jp/map4_engine_en). They currently provide a trial license for Autoware users free of charge.
+
+### Creating a vector map
+
+A vector map is required for route planning, traffic light detection, and predicting the trajectories of other vehicles and pedestrians.
+
+The specifications of an Autoware-compatible vector map are as follows:
+
+- A map must cover the entire operational area of the vehicle and should include an additional buffer zone of at least 200 m in all directions.
+- A map is in [Lanelet2](https://github.com/fzi-forschungszentrum-informatik/Lanelet2) format with[additional modifications](https://github.com/autowarefoundation/autoware.universe/blob/main/map/lanelet2_extension/docs/lanelet2_format_extension.md).
+- A map must contain the shape and position information of lanes, traffic lights, stop lines, crosswalks, parking spaces, and parking lots.
+- Each lanelet must contain information regarding its right of way, speed limit, traffic direction, and associated traffic lights, stop lines, and traffic signs.
+- Unless it is the end/beginning of a road, each lanelet must be correctly connected to its predecessor, successors, left neighbor, and right neighbor for routing planning to be working.
+
+The easiest way to create a vector map that is compatible with Autoware is to use [Vector Map Builder](https://tools.tier4.jp/vector_map_builder/), a web-based tool provided by [TIER IV, Inc.](https://www.tier4.jp/) Vector Map Builder allows you to directly create lanes and add additional regulatory elements such as stop signs or traffic lights in the correct format required by Autoware while using a point cloud map as a reference.
+
+Technically, you can also use open-source software such as [JOSM](https://josm.openstreetmap.de/) to create Lanelet2 primitives, including point and linestring, and save the map as an OSM file. However, various modifications, e.g., adding mandatory tags and relations, must be done manually to make the map compatible with Autoware. This process can be very tedious and time-consuming. Another open-source option worth mentioning is [MapToolbox](https://github.com/autocore-ai/MapToolbox), a plugin for [Unity](https://unity.com/) designed to create a Lanelet2 map for Autoware.
```

Comment:
- Since MapToolbox seems to be the better OSS option, I think we should mention that first and put JOSM at the bottom.
- I also simplified the description of JOSM, as I do not think we need to go into details about why JOSM is bad.

```suggestion
For open-source software options, [MapToolbox](https://github.com/autocore-ai/MapToolbox) is a plugin for [Unity](https://unity.com/) specifically designed to create Lanelet2 maps for Autoware.

Although [JOSM](https://josm.openstreetmap.de/) is another open-source tool that can be used to create Lanelet2 maps, be aware that a number of modifications must be done manually to make the map compatible with Autoware. This process can be tedious and time-consuming and so use of JOSM is not recommended.
```